app/i18n/en/install.php

```diff
@@ -55,15 +55,15 @@
 			'ok' => 'You have the cURL library.',
 		),
 		'data' => array(
-			'nok' => 'Check permissions on the <em>%1$s</em> directory for <em>%2$s</em> user. The HTTP server must have write permission.',
+			'nok' => 'Check permissions on the <em>%s</em> directory (Owner: %s, Group: %s, Rights: %s).<br />The HTTP server (User name: %s) must have write permissions.',
```

Comment:
I am not so found of having so many duplicate strings. Let's make a little refactoring for having a single copy (both for `nok` and `ok`) with the folder name as parameter. We can use the `data` key.